src/basic/env-util.c

```diff
@@ -784,3 +785,42 @@ int setenv_systemd_exec_pid(bool update_only) {
 
         return 1;
 }
+
+int getenv_path_list(const char *name, char ***ret_paths) {
+        _cleanup_strv_free_ char **l = NULL;
+        const char *e;
+        char **p;
+        int r;
+
+        assert(name);
+        assert(ret_paths);
+
+        e = secure_getenv(name);
+        if (!e)
+                return 0;
```

Comment:
initialize the return param to NULL please in this case. i.e. we have this guarantee: if we return error (i.e. < 0) then don#t touch any ret params, but if we succeed (and this is a form of success here) then always initialize. Since we have no data here, initialize to NULL

Reply:
(so the comment here was written before I realized that this is actually both an input + output param... If this is changed to be output only, my comment still applies, but otherwise it's of course doesn't apply